UefiCpuPkg/SecCore/SecCore.inf

```diff
@@ -78,6 +78,7 @@
 
 [Pcd]
   gUefiCpuPkgTokenSpaceGuid.PcdPeiTemporaryRamStackSize  ## CONSUMES
+  gUefiCpuPkgTokenSpaceGuid.PcdPageTableInPermanentRamInitSize  ## CONSUMES

```

Comment:
The PCD name is a bit strange.
I will think about it.

Reply:
How about PcdMaxMappingAddressBeforeTempRamExit?
Do we need another PCD PcdMinMappingAddressBeforeTempRamExit for further performance speed up?